ksql-udf/src/main/java/io/confluent/ksql/function/udf/UdfDescription.java

```diff
@@ -59,7 +59,7 @@
    *
    * @return function author.
    */
-  String author() default "";
+  String author() default "built in";
```

Comment:
This is dangerous as it makes it more likely, IMHO, that other people implementing this will forget to change the author.